src/porepy/fracs/importer.py

```diff
@@ -24,16 +24,18 @@ def from_csv(f_name, mesh_kwargs, domain=None, pause=False, **kwargs):
 
     Returns:
         gb: grid bucket associated to the configuration.
+        domain: if the domain is not given as input parameter, the bounding box
+        is returned.
 
     """
     pts, edges = fractures_from_csv(f_name, **kwargs)
+    f_set = np.array([pts[:, e] for e in edges.T])
 
     # Define the domain as bounding-box if not defined
     if domain is None:
         overlap = kwargs.get('domain_overlap', 0)
         domain = _bounding_box(pts, overlap)
-
-    f_set = np.array([pts[:, e] for e in edges.T])
+        return meshing.simplex_grid(f_set, domain, **mesh_kwargs), domain
```

Comment:
Not sure about the context here, but it seems lines 38 and 40 returns a different number of parameters. Is this on purpose, and is it wise?